Source/Scene/Cesium3DTile.js

```diff
@@ -187,10 +212,8 @@ define([
          */
         this.parentPlaneMask = 0;
 
-        this._debugBox = undefined;
-        this._debugcontentBox = undefined;
-        this._debugOrientedBoundingBox = undefined;
-        this._debugContentsOrientedBoundingBox = undefined;
```

Comment:
This seems like the right approach to me. We will have to modify `Cities.html` to remove some of the extra buttons, but that can be a later commit.